simtools/applications/simulate_showers_for_trigger_rates.py

```diff
@@ -107,9 +120,29 @@ def _parse(label=None, description=None):
         required=True,
     )
     config.parser.add_argument("--nruns", help="Number of runs", type=int, default=100)
+    config.parser.add_argument(
+        "--run_number", help="Run number of the starting run", type=int, default=1
+    )
     config.parser.add_argument("--nevents", help="Number of events/run", type=int, default=100000)
     config.parser.add_argument("--zenith", help="Zenith angle in deg", type=float, default=20)
     config.parser.add_argument("--azimuth", help="Azimuth angle in deg", type=float, default=0)
+    config.parser.add_argument("--view_cone", help="View cone in deg", type=float, default=10)
+    config.parser.add_argument(
+        "--scatter_x", help="Scatter distance (X axis) in m", type=float, default=1500
+    )
+    config.parser.add_argument(
+        "--scatter_y", help="Scatter radius  (Y axis) in m", type=float, default=0
+    )
+    config.parser.add_argument(
+        "--num_use", help="Number of use for each shower", type=int, default=20
```

Comment:
The number '20' come where? It seems to be quite high (but maybe good for trigger calculations?) Don't know.

Reply:
I kept the values that were there before. It seems quite high but it indeed helps with the trigger rate estimate. On the other hand, the estimate of the trigger rate uncertainty is biased the higher the num_use is. To find a compromise, I reduced it now to 10.

Reply:
Good. Thanks.